charts/dvpe-deployment-gloo/templates/secret.yaml

```diff
@@ -2,58 +2,65 @@
 
 {{- if .Values.externalSecrets.service.key }}
 ---
-apiVersion: 'kubernetes-client.io/v1'
+apiVersion: 'external-secrets.io/v1beta1'
 kind: ExternalSecret
 metadata:
   name: {{ $serviceName }}-service-secrets
   namespace: {{ .Release.Namespace }}
 spec:
-  backendType: secretsManager
-  {{- if .Values.externalSecrets.service.roleArn }}
-  roleArn: {{ .Values.externalSecrets.service.roleArn }}
-  {{- end}}
+  secretStoreRef:
+    kind: SecretStore
+    name: {{ .Values.externalSecrets.service.secretstoreName | default (printf "external-secret-store-%s" .Release.Namespace) }}
+  target:
+    name: {{ $serviceName }}-service-secrets
   dataFrom:
-    - {{ .Values.externalSecrets.service.key }}
+  - extract:
+      key: {{ .Values.externalSecrets.service.key }}
+  refreshInterval: {{ .Values.externalSecrets.service.refreshInterval | default "15m" }}
 {{- end }}
 
 {{- if and .Values.externalSecrets.oauth2.key .Values.gloo.authConfig.spec.configs.oauth.clientId }}
 ---
-apiVersion: 'kubernetes-client.io/v1'
+apiVersion: 'external-secrets.io/v1beta1'
 kind: ExternalSecret
 metadata:
   name: {{ $serviceName }}-oidc-secrets
   namespace: {{ .Release.Namespace }}
 spec:
-  backendType: secretsManager
-  {{- if .Values.externalSecrets.service.roleArn }}
-  roleArn: {{ .Values.externalSecrets.service.roleArn }}
-  {{- end}}
-  template:
-    metadata:
-      annotations:
-        resource_kind: '*v1.Secret'
+  secretStoreRef:
+    kind: SecretStore
+    name: {{ .Values.externalSecrets.service.secretstoreName | default (printf "external-secret-store-%s" .Release.Namespace) }}
```

Comment:
again, this can be a fixed value since the name of the SecretStore will be generated